src/main/java/io/github/cottonmc/resources/CommonResources.java

```diff
@@ -23,6 +23,12 @@ public static void initialize() {
         builtinMetal("silver", BlockSuppliers.IRON_TIER_ORE,  MACHINE_AFFIXES);
         builtinMetal("lead",   BlockSuppliers.IRON_TIER_ORE,  MACHINE_AFFIXES);
         builtinMetal("zinc",   BlockSuppliers.STONE_TIER_ORE);
+        builtinMetal("aluminum", BlockSuppliers.IRON_TIER_ORE);
+        builtinMetal("cobalt", BlockSuppliers.IRON_TIER_ORE);
+        builtinMetal("tin", BlockSuppliers.STONE_TIER_ORE);
+        builtinMetal("tungsten", BlockSuppliers.IRON_TIER_ORE);
+
+        builtinMetal("uranium",   BlockSuppliers.IRON_TIER_ORE);
```

Comment:
Revert this. We're considering adding rods and capsules for RADIOACTIVE_AFFIXES.

Reply:
Would be happy to, however the reasoning behind that move was it was configured originally by the builtinItems() method, which does not appear to do any of the registration for the ore iteself only the item like pieces. To me that looked like a bug since you had an ore texture for uranium, but i could not find how you all were actually registering all the stuff associated with it. I very well could have missed something though. It may be that you might need a second set of registrations for the additional radioactive type items, or do you still wish me to revert?  

When I get back home from work I will look into kicking up a game with this mod in and play around, will allow me to more easily see what is going on with things, less likely for me to overlook things.

Reply:
I'll take a look later too, I thought it did do block stuff.